### DISCUSSION TITLE
Do I need internet support for the first launch after installation?

### DISCUSSION THREAD

**Discussion by z1mpleee** [open-webui/open-webui#1297] [General]
# Bug Report

Do I need internet support for the first launch after installation? 

## Description

I installed via Docker in an environment without internet access. When I open the web, it shows a blank page, and in the console, I see the error reported as "Uncaught (in promise) TypeError: S.at is not a function".

![WX](https://github.com/open-webui/open-webui/assets/45474697/846bdc16-64d2-400c-a094-84d54fa18274)

**Comment by Darin755** [comment]
I do not believe you do (I have not tested it)

**Comment by z1mpleee** [comment]
Once I connect to the internet, I can use it normally, but when I switch the server back to the intranet environment, I encounter a blank page again. Is there any way to use it offline?

**Reply by Darin755** [reply]
I was able to use it offline when I was traveling. How is your install setup?

**Reply by z1mpleee** [reply]
I launched Docker with the `--network=host` parameter.

**Comment by changchiyou** [comment]
> When I open the web, it shows a blank page ...

@z1mpleee Could you please inspect your network activity using https://developer.chrome.com/docs/devtools/network#load? 

If the last entry shows a request for `http://your.openwebui.server/ollama/api/tags` and is stuck with a `pending` status, it's likely that there's an error in your ollama installation.

**Reply by z1mpleee** [reply]
所有请求都正常响应

**Reply by z1mpleee** [reply]
![WX](https://github.com/open-webui/open-webui/assets/45474697/4ba2733d-8cb7-46c2-8a6b-adcc7f0acc67)

**Reply by tjbck** [reply]
@z1mpleee You might want to try using the latest version of chrome if you aren't already!

**Reply by z1mpleee** [reply]
After switching to the latest version of Chrome, it works fine now. Thank you!

**Reply by changchiyou** [reply]
@tjbck, could [this issue](https://github.com/open-webui/open-webui/discussions/1297#discussioncomment-8911449) possibly be caused by the same reason, namely, not using the latest version of Chrome?

> ... If the last entry displays a request for http://your.openwebui.server/ollama/api/tags and remains stuck with a pending status, it's probable that there's an error in your Ollama installation.

**Comment by Darin755** [comment]
I got a white screen on sign in when the Ollama env var was an unaccessible IP address.
